libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -1613,6 +1615,14 @@ bool RC_Channel::do_aux_function(const aux_func_t ch_option, const AuxSwitchPos
         AP::ahrs().request_yaw_reset();
         break;
 
+    case AUX_FUNC::AHRS_TYPE: {
+#if HAL_NAVEKF3_AVAILABLE && HAL_EXTERNAL_AHRS_ENABLED
+        AP::ahrs().set_ekf_type(ch_flag==AuxSwitchPos::HIGH? 11 : 3);
```

Comment:
Would it make sense to create an `enum class` in AP_AHRS.h` and use it here instead of magic numbers?
https://github.com/ArduPilot/ardupilot/blob/60ceaec9012a5e86148f2ed77232bb6c252e94ce/libraries/AP_AHRS/AP_AHRS.cpp#L147C3-L147C71

Reply:
yes, done